docs/URL-Checkers.md

```diff
@@ -0,0 +1,25 @@
+# URL Checkers
```

Comment:
Checkers are already mentioned here: https://github.com/cakephp/authentication/blob/master/docs/Authenticators.md#url-checkers

You may want to link this from there or remove checker docs from Authenticators page.